docs/Manual/04 Fusing.md

```diff
@@ -38,11 +38,11 @@ def _():
 
 schedule_simple = nest_simple.create_schedule()
 ```
-Note that each iteration in `schedule_simple` operates on all three arrays at once. Imagine that there is some computational advantage to operating on only two arrays at a time. For example, say that operating on all three arrays simultaneously creates excessive pressure on the computer's memory caches, which could hurt performance.
+Note that each iteration in `schedule_simple` executes simultaneously on all three arrays. However, there can be a case where concurrent operation on these arrays creates unbridled pressure on the computer’s cache, resulting in lower performance. In such a case, simultaneous operation on two arrays instead of three has a computational advantage.
```

Comment:
"excessive" is preferred here, less strong than "unbridled"

(The latter makes it sound much more serious than it may be.)